autonomy/cli/build_images.py

```diff
@@ -42,15 +42,17 @@
     type=click.Path(dir_okay=True),
     help="Path to service dir.",
 )
+@click.option("--version", type=str, help="Specify tag version for the image.")
 @click.option("--dev", is_flag=True, help="Build development image.", default=False)
 @click.option("--pull", is_flag=True, help="Pull latest dependencies.", default=False)
 def build_image(
     agent: Optional[PublicId],
     service_dir: Optional[Path],
     pull: bool = False,
     dev: bool = False,
+    version: Optional[str] = None,
 ) -> None:
-    """Build image using skaffold."""
+    """Build runtime images for autonomous agents."""
```

Comment:
@Adamantios updated the help message as per your suggestion

Reply:
Still not helpful enough. We should clarify that it can either build a deployment of a service or an agent image.

We may also add a help message for the agent argument because it currently does not show up as an option:

```
Usage: autonomy build-image [OPTIONS] [PUBLIC_ID_OR_HASH]

  Build runtime images for autonomous agents.

Options:
  --service-dir PATH  Path to service dir.
  --version           Specify tag version for the image.
  --dev               Build developement image.
  --pull              Pull latest dependencies.
  --help              Show this message and exit.
```

Reply:
We do that in the documentation of the CLI tool and `autonomy build-image [OPTIONS] [PUBLIC_ID_OR_HASH]` specifies that public id can be used to build images